include/bluetooth/bluetooth.h

```diff
@@ -637,6 +637,48 @@ struct bt_le_per_adv_param {
 						 BT_GAP_ADV_FAST_INT_MAX_2, \
 						 NULL)
 
+/** Non-connectable extended advertising with private address */
+#define BT_LE_EXT_ADV_NCONN BT_LE_ADV_PARAM(BT_LE_ADV_OPT_EXT_ADV, \
+					    BT_GAP_ADV_FAST_INT_MIN_2, \
+					    BT_GAP_ADV_FAST_INT_MAX_2, NULL)
+
```

Comment:
Now that you are already adding these new APIs, maybe you should also add `BT_LE_EXT_ADV_NCONN_CODED`?
Maybe also apply the same pattern to `BT_LE_EXT_ADV_NCONN_NAME` and `BT_LE_EXT_ADV_NCONN_IDENTITY`

Reply:
I now have included the sample to sync with the periodic advertising, will add the defines for coded PHY and push the changes once I am done testing.